EECS3311-PD1/src/main/resources/fxml/MainPane.fxml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.layout.*?>
+
+<?import UI.TopMenuView?>
+<BorderPane xmlns="http://javafx.com/javafx"
+            xmlns:fx="http://javafx.com/fxml"
+            fx:controller="UI.MainPane"
+            prefHeight="400.0" prefWidth="600.0"
+            >
+
+    <top>
+        <TopMenuView fx:id="topMenu" onCountryOrYearsSelected="#countryOrYearsSelected" alignment="TOP_CENTER"/>
+    </top>
+
+</BorderPane>
```

Comment:
Don't need this file in this PR, will create a MainPane and integrate things together in another ticket